src/transformers/generation/logits_process.py

```diff
@@ -345,16 +345,48 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 
 class TypicalLogitsWarper(LogitsWarper):
     r"""
-    [`LogitsWarper`] that performs typical decoding. See [Typical Decoding for Natural Language
-    Generation](https://arxiv.org/abs/2202.00666) for more information.
+    [`LogitsProcessor`] that applies typical warping techniques to the logits during text generation. It can be used to
```

Comment:
```suggestion
    [`LogitsWarper`] that applies typical warping techniques to the logits during text generation. It can be used to
```
(it is a `LogitsWarper`, which is a special case of a `LogitsProcessor` that is only applicable when `do_sample==True`)